### DISCUSSION TITLE
Плажки заданий

### DISCUSSION THREAD

**Discussion by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#151] [Ideas]
Сделал более читаемые и аккуратные плажки для заданий
Если понравится сделаю остольное 
![Screenshot_2023-10-27-22-20-58-109_com pixel gun combats (1)](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/146564992/2d18ca77-33a0-4059-9cec-5e5cf77447fa)
![pixelcut-export-1697892674547](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/146564992/835705bb-a06d-4c28-bdae-66213e6bfdea)

**Comment by kkohno** [comment]
спасибо, посмотрим
